### DISCUSSION TITLE
Пару вопросов

### DISCUSSION THREAD

**Discussion by imcrazyhoudini** [OGSR/OGSR-Engine#431] [Q&A]
Давно перенёс свой мод на данный движок, но только сейчас решил заняться всеми красными строчками в консоли.
1. [04.08.22 16:52:34.689] ! Version conflict in shader 'models\lightplanes_00'
и подобное. shaders.xr стоит с огср, правки в него никакие не вносились. Как править?
2. [04.08.22 16:55:03.201] ! Unknown command:  r2_sun_shafts
[04.08.22 16:55:03.201] ! Unknown command: r__supersample
В консоли если вписать подобное можно действительно обнаружить, что таких команд нет.
Значит, достаточно удалить их с rspec_***.ltx? (я так и сделал, но вдруг можно было исправить).
3. Что могут значить подобные строчки?:
[04.08.22 17:03:10.544] * [CSpaceRestrictionHolder::register_restrictor]: change restrictor_type of zone_burning_fuzz_0001 to eRestrictorTypeNone because border().empty()
[04.08.22 17:03:10.545] * [CSpaceRestrictionHolder::register_restrictor]: change restrictor_type of zone_burning_fuzz_0002 to eRestrictorTypeNone because border().empty()
[04.08.22 17:03:10.546] * [CSpaceRestrictionHolder::register_restrictor]: change restrictor_type of zone_burning_fuzz_0003 to eRestrictorTypeNone because border().empty()
4. Есть у меня подобная проблема описанная здесь [https://github.com/OGSR/OGSR-Engine/discussions/395](url).
У меня оно вызвано тем, что НПС использует новые анимации (в инете их можно найти загуглив "анимации для толстых сталкеров". Придётся их убрать получается?
5. Почему вокруг аномалий не спавнятся артефакты?
6. Команда g_music_tracks (может с ошибками написал) отвечает за то, будет ли играть амбиент на локации?

Сорян за поток вопросов, но сам разобраться не могу.

**Comment by joye-ramone** [comment]
2 - просто удалить
3 - неправильный настройки рестораторов, не заданы границы при этом тим не none
5 - настройки gamedate нужно смотреть. двиг умеет спаунить арты если это настроено

**Reply by imcrazyhoudini** [reply]
> 
3. вот подскажи конкретнее:
https://ibb.co/xhCrPMQ
что не так здесь?

5 - до переезда на огср всё было норм, настройки те же, то бишь:
artefacts		= af_electric_shar, 0.05, af_bulizh_blue, 0.95
и в каждой секции этой аномалии так же есть

**Reply by joye-ramone** [reply]
3 - не подскажу. я в SDK никогда не заглядывал )
5 spawn_blowout_artefacts = on

**Reply by imcrazyhoudini** [reply]
5. ;рождение артефактов во время срабатывания

это если коснуться аномалии то будет спавн артов, а не как в дефолтном тч?

**Reply by joye-ramone** [reply]
Не знаю как было в дефолтном ТЧ
просто так арты не генерируются движком
описание есть тут https://github.com/OGSR/OGSR-Engine/wiki/%D0%9F%D0%BE%D0%B4%D1%80%D0%BE%D0%B1%D0%BD%D0%BE%D0%B5-%D0%BE%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D0%B5-%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D0%B5%D0%BD%D0%B8%D0%B9

**Reply by imcrazyhoudini** [reply]
> Не знаю как было в дефолтном ТЧ просто так арты не генерируются движком описание есть тут https://github.com/OGSR/OGSR-Engine/wiki/%D0%9F%D0%BE%D0%B4%D1%80%D0%BE%D0%B1%D0%BD%D0%BE%D0%B5-%D0%BE%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D0%B5-%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D0%B5%D0%BD%D0%B8%D0%B9

ну тут ничего на счёт этого не сказано

в тч просто арты рандомно спавнились при старте игры (если не ошибаюсь), и в электрах вроде ночью спавнились (могу путать)

тут ничего такого нет, так что артефактов нигде не найти

**Reply by joye-ramone** [reply]
движок не спаунит сам рендомно арты возле аномалий. есть только логика описанная в wiki
